modules/cache/outputs.tf

```diff
@@ -0,0 +1,19 @@
+output "id" {
+  value       = var.cache_store == "memcached" ? module.memcached[0].cluster_id : var.cache_store == "redis" ? module.redis[0].id : "none"
+  description = "Cache store ID"
+}
+
+output "endpoint" {
+  value       = var.cache_store == "memcached" ? module.memcached[0].cluster_address : var.cache_store == "redis" ? module.redis[0].endpoint : "none"
+  description = "Cache store address"
+}
+
+output "port" {
+  value       = var.cache_store == "memcached" ? "11211" : var.cache_store == "redis" ? module.redis[0].port : "0"
```

Comment:
Should expose the module port output for memcached

Reply:
composse `memcached` module doesn't output port, so we have to hard-code it.

Reply:
In that case how about just setting it to the value of the port variable?

Reply:
Oh we're not asking for the port.. In that case, is this output necessary? It will only ever use the default.